python/Ganga/Core/GangaThread/WorkerThreads/__init__.py

```diff
@@ -2,6 +2,9 @@
 _global_queues = None
 _queues_interface = None
 
+def getQueues():
+    return _global_queues
+
```

Comment:
I'm not sure we need a getter function here. I think that just renaming `_global_queues` to `global_queues` and importing that would be sufficient.